desktop/i.html

```diff
@@ -656,7 +659,7 @@
 			})
 			.catch(error => {
 				console.error("error", error);
-				document.body.innerHTML = "";
+				// document.body.innerHTML = "";
```

Comment:
Commented this because in the current framework, it will just have whitespace. Now, it will show the loader. We can add it back once we go for the new Landing Page